sphinx-docs/Plugin-library.md

```diff
@@ -165,6 +165,23 @@ When this plugin has been loaded, CALDERA will start the HAProxy service on the
 
 CALDERA will **only** be available at https://[YOUR_IP]:8443 when using this plugin. All deployed agents should use the correct address to connect to CALDERA. 
 
+**Warning:** This plugin uses a default self-signed ssl certificate and key which is insecure. It is highly recommended that you generate your own before using the plugin to increase the safety of the system. See directions below.
+
+#### Using your own self-signed certificate
+In order to use this plugin securely, you need to generate your own self-signed certificate.
+The following directions explain how to generate and install your own certificate on a Linux or macOS system. 
+
+##### Directions:
+*Note: OpenSSL must be installed on your system*
+1. In the root CALDERA directory, navigate to `plugins/ssl`.
+2. Remove the file `conf/insecure_certificate.pem`.
```

Comment:
I'm inclined to leave this file alone (not delete) since it's included in .git .  That shouldn't hurt anything so long as the haproxy config references a different path and you make the requested changed in https://github.com/mitre/ssl/pull/24